app/Http/Controllers/RecentController.php

```diff
@@ -12,37 +12,30 @@ class RecentController extends Controller
     public function recent()
     {
         $user = Auth::user();
-        if(Auth::check()){
-            $recentBooks = Recent::where('user_id', Auth::user()->user_ID)->orderBy('updated_at', 'desc')->get();
+        $recentBooks = $user->recent()->orderBy('updated_at', 'desc')->get();
```

Comment:
you can achieve similar result if you use the fluent relatinship select 

The foreach below - this is the n+1 problem in sql - a very common and dangerous pitfall of performance

https://planetscale.com/blog/what-is-n-1-query-problem-and-how-to-solve-it